app/models/agents/twitter_user_agent.rb

```diff
@@ -81,10 +88,14 @@ def starting_at
     def include_retweets?
       interpolated[:include_retweets] != "false"
     end
+    
+    def exclude_replies?
+      interpolated[:exclude_replies] != "false"
```

Comment:
Since the current default is `false`, it would be best if it stayed that way for existing users.  Anyone who updates will have `interpolated[:exclude_replies]` be `nil`, which is `interpolated[:exclude_replies] != "false"`, and so this method will return `true`, which is not the current default.  I'd propose we do

```ruby
def exclude_replies?
  boolify(interpolated[:exclude_replies]) || false
end
```